### DISCUSSION TITLE
A javascript error occured in the main process

### DISCUSSION THREAD

**Discussion by Yesitzme** [OhMyGuus/BetterCrewLink#3] [General]
![crewlink](https://user-images.githubusercontent.com/76435834/102906089-6d904a80-4474-11eb-8b4e-e6a27ae7b454.png)

**Comment by MatadorProBr** [comment]
This is probably because you are using Windows 32 bits and in this version of CrewLink-Beta that is now called BetterCrewLink it does not work with people who had a 32-bit PC, but now in this new version this has been fixed, so I recommend that you [download the new one version](https://github.com/OhMyGuus/BetterCrewLink/releases/latest) (I know I'm saying this 1 year after the discussion started but he'll see it today and try to download it again)
